fire_atlas_editor/lib/screens/editor_screen/editor_screen.dart

```diff
@@ -1,27 +1,25 @@
-import 'package:slices/slices.dart';
+import 'package:fire_atlas_editor/screens/editor_screen/widgets/preview.dart';
+import 'package:fire_atlas_editor/screens/editor_screen/widgets/selection_canvas/selection_canvas.dart';
+import 'package:fire_atlas_editor/screens/editor_screen/widgets/selection_list.dart';
+import 'package:fire_atlas_editor/screens/editor_screen/widgets/toolbar.dart';
+import 'package:fire_atlas_editor/screens/widgets/scaffold.dart';
+import 'package:fire_atlas_editor/store/store.dart';
+import 'package:fire_atlas_editor/widgets/button.dart';
 import 'package:flutter/material.dart';
-
-import './widgets/toolbar.dart';
-import './widgets/selection_canvas/selection_canvas.dart';
-import './widgets/selection_list.dart';
-import './widgets/preview.dart';
-
-import '../widgets/scaffold.dart';
-import '../../widgets/button.dart';
-import '../../store/store.dart';
+import 'package:slices/slices.dart';
 
 class EditorScreen extends StatelessWidget {
+  const EditorScreen({Key? key}) : super(key: key);
+
   @override
-  Widget build(ctx) {
+  Widget build(BuildContext ctx) {
     final _store = SlicesProvider.of<FireAtlasState>(ctx);
     if (_store.state.currentAtlas == null) {
-      // TODO improve this
-      // To the Erick from the past: Improve what?
```

Comment:
Lol

Reply:
if past erick doesnt remember what past past erick meant, what chances can current or future erick possibly have... might as well remove the todo :P